src/data_source/storage/sql.rs

```diff
@@ -1762,6 +1915,25 @@ mod test {
         connect(true, migrations).await.unwrap();
     }
 
+    #[async_std::test]
+    async fn test_current_migrations() {
```

Comment:
I believe `test_migrations` already tests whatever is in the `migrations` directory